flutter/lib/src/web/script_loader/html_script_dom_api.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+import 'dart:html';
+
+Future<void> loadScript(String src, String? integrity) {
+  final completer = Completer<void>();
+  final script = ScriptElement()
+    ..src = src
+    ..crossOrigin = 'anonymous'
+    ..onLoad.listen((_) => completer.complete())
+    ..onError.listen((event) => completer.completeError('Failed to load $src'));
+
+  if (integrity != null) {
+    script.integrity = integrity;
+  }
+
+  document.head?.append(script);
```

Comment:
`h`: We should ensure that the Sentry JS SDK gets added before all other scripts because otherwise, our JS SDK can't report problems with the scripts loaded before it; see https://docs.sentry.io/platforms/javascript/#install.